app/models/qdm/patient.rb

```diff
@@ -0,0 +1,143 @@
+class QDM::Patient
+  include Mongoid::Document
+  field :birth_datetime, type: DateTime
+  field :qdm_version, type: String, default: '5.3'
+  field :ethnicity, type: QDM::Code
+  field :race, type: QDM::Code
+  field :sex, type: QDM::Code
```

Comment:
Same questions that I asked above on the Javascript model. Do we need to add 'payer' and add an additional function for the 'patient_characteristic' category?

Reply:
See previous reply ☝️